src/fides/lib/oauth/api/routes/user_endpoints.py

```diff
@@ -191,8 +196,10 @@ def user_login(
             created_at=datetime.utcnow(),
         )
     else:
-        user_check: Optional[FidesUser] = FidesUser.get_by(
-            db, field="username", value=user_data.username
+        user_check = (
+            FidesUser.query(db)
+            .filter(func.lower(FidesUser.username) == user_data.username.lower())
+            .first()
```

Comment:
Looks like we could DRY this up and share it with L75!

Reply:
Now that citext is being used I reverted this query to the original.